exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
-var numberOfStudents = 15;
+var numberOfStudents = 15
+numberOfStudents = 15/23*100;
 var numberOfMentors = 8;
+numberOfMentors = 8/23*100;
+console.log("Percentage students: " + numberOfStudents.toFixed() + "%");
+console.log("Percentage mentors: " + numberOfMentors.toFixed() + "%");
```

Comment:
Hi  Elle I am just reviewing your code. very nice work it looks like your getting the hang of javascript